### PR TITLE
Add Signature and bitmap of witnesses to the block header

### PR DESCRIPTION
**Pull request by rtsisyk**
Just add to the block header. Validation is not implemented yet.

Needed for #42

### CODE REVIEW

File: crypto/src/pbc/secure.rs [hunk 1]
```diff
@@ -200,10 +200,16 @@ impl Neg for Zr {
 pub struct G1([u8; G1_SIZE_FR256]);
 
 impl G1 {
-    pub fn new() -> G1 {
+    /// Create a new point which binary representation consists of all zeros.
+    pub fn null() -> G1 {
```
Comment (vldm) [inline crypto/src/pbc/secure.rs]: may be use `zero` name like in other crypto types ?
```suggestion
    pub fn zero() -> G1 {
```

File: crypto/src/pbc/secure.rs [hunk 2]
```diff
@@ -673,6 +679,17 @@ impl PartialEq for PublicSubKey {
 pub struct Signature(G1);
 
 impl Signature {
+    /// Create a new point which binary representation consists of all zeros.
+    pub fn null() -> Signature {
```
Comment (vldm) [inline crypto/src/pbc/secure.rs]:
```suggestion
    pub fn zero() -> G1 {
```

File: blockchain/src/block.rs [hunk 3]
```diff
@@ -34,9 +35,12 @@ use stegos_crypto::curve1174::fields::Fr;
 use stegos_crypto::curve1174::G;
 use stegos_crypto::hash::{Hash, Hashable, Hasher};
 use stegos_crypto::pbc::secure::PublicKey as SecurePublicKey;
+use stegos_crypto::pbc::secure::Signature as SecureSignature;
+
+pub const WITNESSES_MAX: usize = 128;
```
Comment (vldm) [inline blockchain/src/block.rs]: why we have this limit?

Comment (rtsisyk) [reply blockchain/src/block.rs]: Just some reasonable number. Can increase to 512 or more.

File: crypto/src/pbc/secure.rs (hunk 1 above)
Comment (rtsisyk) [reply crypto/src/pbc/secure.rs]: I don't like both variants. zero() can be perceived as zero point on the curve... I think that this problem is caused by bad design of block header. Why we want to store "empty" signature and then change it?

File: blockchain/src/block.rs (hunk 3 above)
Comment (vldm) [reply blockchain/src/block.rs]: Have technical limit, or it's just for testing?

**Comment by rtsisyk** [comment]
I'm integrating this pull request to the main branch #281

**Pull request closed without merging**